### PR TITLE
Updated to support IISExpress and add new test cases for the addition…

### PR DESCRIPTION
**Pull request by jhkimnew**
@shirhatti @pan-wang

### CODE REVIEW

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj [hunk 1]
```diff
@@ -7,11 +7,20 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
 
+  <PropertyGroup>
+      <!--<VSTestCLIRunSettings>$(MSBuildProjectDirectory)\.runsettings</VSTestCLIRunSettings>-->
```
Comment (shirhatti) [inline test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: Remove this comment

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj [hunk 2]
```diff
@@ -7,11 +7,20 @@
...
+      <!--<VSTestCLIRunSettings>.\.runsettings</VSTestCLIRunSettings>-->
+      <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
```
Comment (shirhatti) [inline test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: Move this to repo.props?

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj [hunk 3]
```diff
@@ -41,6 +49,18 @@
     <Reference Include="System.ServiceProcess" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
+    <Reference Include="System.Management.Automation" />
+  </ItemGroup>
+
+  <!--
```
Comment (shirhatti) [inline test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: Cleanup

**Comment by shirhatti** [comment]
Can you please add a script to restore my apphost.config 😄 after running these tests

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj (hunk 2 above)
Comment (jhkimnew) [reply test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: It seems that we should use common.props to make it work when running test manually.

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj (hunk 1 above)
Comment (jhkimnew) [reply test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: Done

File: test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj (hunk 3 above)
Comment (jhkimnew) [reply test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj]: Done

**Comment by jhkimnew** [comment]
@shirhatti @pan-wang There are some update after CR.

**Pull request merged**